ros2_ws/install/slu_smart/share/ament_index/resource_index/packages/slu_smart

```diff
@@ -0,0 +1 @@
+/home/pascal/ros2_ws/build/slu_smart/resource/slu_smart
```

Comment:
I'm seeing lots of paths like this, with a hard coded absolute path with a username